webpack.common.config.js

```diff
@@ -88,7 +88,7 @@ module.exports = Merge.smart({
                                                'program_enrollments/index.jsx',
             ProgramEnrollmentsInspectorPage: './lms/djangoapps/support/static/support/jsx/' +
                                                'program_enrollments/inspector.jsx',
-            PasswordResetConfirmation: './lms/static/js/student_account/components/PasswordResetConfirmation.jsx',
+            PasswordResetConfirmation: '/edx/app/edxapp/edx-platform/themes/adg-edx-theme/edx-platform/adg/lms/static/js/student_account/components/PasswordResetConfirmation.jsx',
```

Comment:
Let's please use COMPREHENSIVE_THEME_DIRS settings. Because path would be different on all of our servers.